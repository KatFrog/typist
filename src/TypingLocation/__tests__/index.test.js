import React from 'react';
import TypingLocation from '../';
import renderer from 'react-test-renderer';

describe('TypingLocation', () => {
  it('renders correctly', () => {
    const location = 10;
    const component = renderer.create(
      <TypingLocation {...{ location }} />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
