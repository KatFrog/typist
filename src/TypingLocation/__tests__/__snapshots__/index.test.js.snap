// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TypingLocation renders correctly 1`] = `
<sup>
  on index 
  10
</sup>
`;
